api/notifications/notifications.go

```diff
@@ -0,0 +1,202 @@
+package notifications
+
+import (
+	"context"
+	"errors"
+	"fmt"
+	"net/http"
+	"strconv"
+	"time"
+
+	"github.com/gorilla/websocket"
+
+	"github.com/Peripli/service-manager/pkg/query"
+	"github.com/Peripli/service-manager/pkg/types"
+	"github.com/Peripli/service-manager/pkg/util"
+	"github.com/Peripli/service-manager/storage"
+
+	"github.com/Peripli/service-manager/pkg/log"
+
+	"github.com/Peripli/service-manager/pkg/web"
+	"github.com/Peripli/service-manager/pkg/ws"
+)
+
+const (
+	lastKnownRevisionHeader     = "last_known_revision"
+	lastKnownRevisionQueryParam = "last_known_revision"
+)
+
+func (c *Controller) handleWS(req *web.Request) (*web.Response, error) {
+	user, _ := web.UserFromContext(req.Context())
```

Comment:
It is best if we do not dismiss the bool here even though it is probably not possible to be false.